### DISCUSSION TITLE
Bootstrap carousel works fine locally, but not on remote server

### DISCUSSION THREAD

**Discussion by LITUATUI** [hbstack/theme-cards#266] [Q&A] [answered]
The [carousel](https://getbootstrap.com/docs/5.3/components/carousel/) I added to the sticky sidebar of [my website](https://pushevs.com) works fine locally with slide and fade effects, but on remote server (Cloudflare pages) it looks strange and seems that images just jump from one to another - like the JS part that smooths the transitions is missing.

It seems to be [this](https://stackoverflow.com/questions/12139670/bootstraps-javascript-works-locally-but-not-when-deployed-to-a-server) issue with the Bootstrap JS file.

I know that some features of Cloudflare like Rocket Loader™ or Auto Minify can mess with JS, but I have them disabled.

This isn't urgent, I'm just curious about why this happens...

**Comment by razonyang** [comment]
Probably the styles get purged by PurgeCSS, will check tonight.

**Reply by LITUATUI** [reply]
Thanks, yes you're right.

I just tested locally with the [production flags](https://hbstack.dev/develop/purgecss/#testing) and it also failed to work properly. So it's not a Cloudflare problem.

By the way, days ago I found that enabling Cloudflare Rocket Loader disables the dark/light theme button, but this seems to be a known issue.

**Comment by razonyang** [comment, marked as answer]
Got sick yesterday, just got time to check this, it should be fixed since `github.com/hbstack/carousel`'s [v0.2.5](https://github.com/hbstack/carousel/releases/tag/v0.2.5).

```sh
hugo mod get github.com/hbstack/carousel@v0.2.5
```

**Reply by LITUATUI** [reply]
It worked, thanks!
